pennylane/interfaces/torch.py

```diff
@@ -199,6 +204,19 @@ def forward(ctx, input_kwargs, *input_):
                         cuda_device = i.get_device()
                         return torch.as_tensor(torch.from_numpy(res), device=cuda_device)
 
+            returns_state = any(
+                [
+                    getattr(op, "return_type", None) is ObservableReturnTypes.State
+                    for op in qnode.ops
+                ]
+            )
+
+            if returns_state and not MIN_VERSION_FOR_STATE:
+                raise ImportError(
+                    "Version 1.6.0 or above of PyTorch must be installed to return the quantum "
+                    "state using the torch interface"
+                )
+
```

Comment:
I know we discussed not having an explicit check for the torch version and mentioning it in the docs - but I feel that given the recentness of the only useable release for complex numbers, an explicit check might help for now.

Reply:
No this is great! In that comment I meant instead not having a specific check for deprecating support for old minor versions.

E.g., if we drop support for Torch 1.3 (and require Torch 1.4+ in the future), we should have checks in the code raising exceptions for Torch <= 1.3.